datahub-web-react/src/app/analyticsDashboard/components/TimeSeriesChart.tsx

```diff
@@ -135,13 +146,71 @@ export const TimeSeriesChart = ({
 
     return (
         <>
+            <GlobalStyle />
             <XYChart
                 accessibilityLabel={chartData.title}
                 width={width}
                 height={height}
                 margin={{ top: MARGIN.TOP, right: MARGIN.RIGHT, bottom: MARGIN.BOTTOM, left: MARGIN.LEFT }}
                 xScale={{ type: 'time' }}
                 yScale={yScale ?? { type: 'linear' }}
+                theme={{
```

Comment:
nice - this is what i'd expect. was the global color override above not enough? 

im just curious why we needed both